Add logrotate example to backend reference (#2387)
sensu/sensu-docs: content/sensu-go/5.19/guides/systemd-logs.md

logrotate -d /etc/logrotate.d/sensu.conf
{{< /highlight>}}

Sensu also offers logging of event data to a separate log file as a commercial feature. See [Sensu backend reference][1] for more information about event logging.

> [1]: ../../reference/backend/#event-logging

Author: ```suggestion
[1]: ../../reference/backend/#event-logging
[2]: ../../getting-started/enterprise/
```

logrotate -d /etc/logrotate.d/sensu.conf
{{< /highlight>}}

> Sensu also offers logging of event data to a separate log file as a commercial feature. See [Sensu backend reference][1] for more information about event logging.

Author: ```suggestion
Sensu also offers logging of event data to a separate log file as a [commercial feature][2].
See the [Sensu backend reference][1] for more information about event logging.
```

logrotate -d /etc/logrotate.d/sensu.conf
{{< /highlight>}}

> Read the [Sensu backend reference][1] for more information about event logging and log rotation, including [examples for systemd and sysvinit][2].

Reviewer: ```suggestion
Sensu also offers logging of event data to a separate log file as a commercial feature. See [Sensu backend reference][1] for more information about event logging.
```

[1]: ../../reference/backend/#event-logging

> [2]: ../../reference/backend/#log-rotation

Reviewer: ```suggestion
```
